pkg/controller/destinationrule/destinationrule_controller.go

```diff
@@ -19,6 +19,7 @@ package destinationrule
 import (
 	"context"
 	"fmt"
+	"kubesphere.io/kubesphere/pkg/models/servicemesh"
```

Comment:
why do we move util package into models? That would makes the controller depend on the models package.

Reply:
There are some other controllers depend on the package `util`, such as application controller、destinationrule controller. The `util` package located under the folder of virtualservice controller . Its path is `pkg/controller/virtualservice/util/util.go`.  I think it's unresonable.

Reply:
change `controller/virtualservice/util` to `controller/util/servicemesh`.